src/main/interface/msp_protocol.h

```diff
@@ -310,6 +310,9 @@
 #define MSP_SET_GPS_RESCUE       225    //in message          GPS Rescues's angle, initialAltitude, descentDistance, rescueGroundSpeed, sanityChecks and minSats
 #define MSP_SET_GPS_RESCUE_PIDS  226    //in message          GPS Rescues's throttleP and velocity PIDS + yaw P
 
+#define MSP_VTX_TABLE            227    //in message          VTX Table
```

Comment:
We don't seem to be using these here - can we allocate them in the pull request that is adding MSP support?